Sources/Purchasing/EntitlementInfos.swift

```diff
@@ -66,31 +65,67 @@ import Foundation
         return self.all == other.all
     }
 
+    // MARK: -
+
+    private let sandboxEnvironmentDetector: SandboxEnvironmentDetector
+
+}
+
+public extension EntitlementInfos {
+
+    /// Dictionary of active ``EntitlementInfo`` objects keyed by their identifiers.
+    /// - Warning: this is equivalent to ``activeInAnyEnvironment``
+    /// - Seealso: ``activeInCurrentEnvironment``
+    @objc var active: [String: EntitlementInfo] {
+        return self.activeInAnyEnvironment
+    }
+
+    /// Dictionary of active ``EntitlementInfo`` objects keyed by their identifiers.
+    /// - Note: these can be active on any environment.
+    /// - Seealso: ``activeInAnyEnvironment``
+    @objc var activeInCurrentEnvironment: [String: EntitlementInfo] {
+        return self.activeInAnyEnvironment.filter { [isSandbox = self.sandboxEnvironmentDetector.isSandbox] in
+            $0.value.isSandbox == isSandbox
+        }
+    }
+
+    /// Dictionary of active ``EntitlementInfo`` objects keyed by their identifiers.
+    /// - Note: this only returns entitlements active in sandbox when queried from the sandbox
+    /// or active in production when queried from production.
```

Comment:
I was confused about how this relates to non-sandbox, maybe an update to the verbiage:
```
    /// - Note: When queried from the sandbox environment, it only returns entitlements active in sandbox.
    /// When queried from production, this will include both production and sandbox.
    
```

Reply:
Much better thanks.

Reply:
Actually yeah the note was completely wrong.

Reply:
Thanks for catching this, the docstrings were backwards!

Reply:
Updated.